server/models/user.js

```diff
@@ -8,7 +8,7 @@ module.exports = (sequelize, DataTypes) => {
      * The `models/index` file will call this method automatically.
      */
     static associate(models) {
```

Comment:
Very well. Put the name of Models here. => {Cohort}

Reply:
thank you Navid, I've changed the name to Cohort.